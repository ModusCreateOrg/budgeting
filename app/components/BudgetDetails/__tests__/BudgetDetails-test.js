import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Connected, { BudgetDetails } from '../';

// mock nested components
jest.mock('../BackButton', () => 'div');
jest.mock('react-router-dom');

describe('Testing BudgetDetails...', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <BudgetDetails />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
