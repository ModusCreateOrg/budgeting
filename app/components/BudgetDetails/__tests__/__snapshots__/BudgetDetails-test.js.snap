// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing BudgetDetails... renders correctly 1`] = `null`;
